src/posix.mak

```diff
@@ -22,9 +22,15 @@ INSTALL_DIR=../../install
 SYSCONFDIR=/etc
 PGO_DIR=$(abspath pgo)
 
-C=backend
-TK=tk
-ROOT=root
+D = ddmd
+
+C=$D/backend
+TK=$D/tk
+ROOT=$D/root
+
+GENERATED = ../generated
+G = $(GENERATED)/$(OS)$(MODEL)
+$(shell mkdir -p $G)
```

Comment:
In the end, the path will be generated/dmd, but until then we will keep a copy of dmd in src/ so that we will have a smooth transition